pipeline/plugin.go

```diff
@@ -119,3 +125,18 @@ const (
 	MatchModeOr      MatchMode = 1
 	MatchModeUnknown MatchMode = 2
 )
+
+// PluginSelector only valid value now is ByNameSelector
```

Comment:
the only valid

Reply:
for now

Reply:
My upper intermediate more rather intermediate :(